application/core/QuestionTypes/ListDropdown/RenderListDropdown.php

```diff
@@ -21,9 +21,26 @@ class RenderListDropdown extends QuestionBaseRenderer
     protected $othertext;
     protected $optCategorySeparator;
     protected $bPrefix;
+
+    /** @var boolean indicates if the question has the 'Other' option enabled */
+    protected $hasOther = false;
+
+    /** @var int the position where the 'Other' option should be placed. Possible values: 0 (Before no answer), 1 (At beginning), 2 (At end), 3 (After specific option)*/
+    protected $otherPosition;
+
+    /** @var string the code of the answer after which the 'Other' option should be placed (if $otherPosition == 3) */
+    protected $answerBeforeOther;
+
+    /** @var boolean indicates if 'Other' option has already been rendered */
+    protected $otherRendered = false;
     
     private $iRowNum = 0;
 
+    const OTHER_POS_BEFORE_NOANSWER = 'default';
```

Comment:
These and other lines seems duplicated in lots of classes? Can we move it out to a trait instead?

Reply:
I will create a new DEV ticket for that

Reply:
https://bugs.limesurvey.org/view.php?id=18489